packages/apps/shopify-api/lib/clients/storefront/__tests__/storefront_client.test.ts

```diff
@@ -136,7 +136,8 @@ describe('Storefront GraphQL client', () => {
       headers: {
         [ShopifyHeader.StorefrontPrivateToken]: session.accessToken,
         [ShopifyHeader.StorefrontSDKVariant]: 'storefront-api-client',
-        [ShopifyHeader.StorefrontSDKVersion]: expect.any(String),
+        // Leaving this to test out if Jest is freaking out over nothing
+        [ShopifyHeader.StorefrontSDKVersion]: '1.0.1',
```

Comment:
This is probably going to fail when we publish a new version of the package, since this will change. I can't tell why it failed with `expect.any(String)` as this is clearly a string (or is it?), but it will have to stay as it was or we won't be able to publish new versions.